dev_tools/scripts/db_integrity_test.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+docker exec db psql -U postgres -d test -c "INSERT INTO local_ega.main(submission_file_path,submission_user,submission_file_extension,status,encryption_method) VALUES('test.c4gh','dummy','c4gh','INIT','CRYPT4GH');"
+
+docker container run --rm -i --name pg-backup --network=host $(docker build -f dev_tools/Dockerfile-backup -q .) /bin/sda-backup --action pg_basebackup
+
+DBCOPY=$(s3cmd -c dev_tools/s3conf ls s3://dumps/ | grep ".tar" | cut -d '/' -f4)
+
+docker container run -v $(pwd)/tmp:/home --rm -i --name pg-backup --network=host $(docker build --build-arg USER_ID=$(id -u) -f dev_tools/Dockerfile-backup -q .) /bin/sda-backup --action pg_bb-restore --name "$DBCOPY"
+
```

Comment:
This is an odd way of doing it since the container will be rebuilt multiple times during the same test.
Preferably the container will be built with a unique name, hash or timestamp or similar, and reused in all the tests.
So the script should build the container unless a name is supplied on execution.

Reply:
Good point, it has been done now